entwatch/ze_mars_escape_xmas

```diff
@@ -0,0 +1,339 @@
+            },
+            {
+                "type": "button",
+                "hammerid": "58355",
+                "event": "OnPressed",
+                "mode": 2,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "button",
+                "hammerid": "58356",
+                "event": "OnPressed",
+                "mode": 2,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },
+    {
+        "name": "Gravity",
+        "shortname": "Gravity",
+        "hammerid": "53036",
+        "message": true,
+        "ui": true,
+        "transfer": true,
+        "color": "Purple",
+        "triggers": [""],
+        "handlers": [
+            {
+                "type": "button",
+                "hammerid": "53032",
+                "event": "OnPressed",
+                "mode": 2,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "button",
+                "hammerid": "58355",
+                "event": "OnPressed",
+                "mode": 2,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "button",
+                "hammerid": "58356",
+                "event": "OnPressed",
+                "mode": 2,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },       
+    {
+        "name": "Electro",
+        "shortname": "Electro",
+        "hammerid": "53008",
+        "message": true,
+        "ui": true,
+        "transfer": true,
+        "color": "Blue",
+        "triggers": [""],
+        "handlers": [
+            {
+                "type": "button",
+                "hammerid": "53004",
+                "event": "OnPressed",
+                "mode": 5,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "button",
+                "hammerid": "58347",
+                "event": "OnPressed",
+                "mode": 5,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "button",
+                "hammerid": "58348",
+                "event": "OnPressed",
+                "mode": 5,
+                "cooldown": 55,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },       
+    {
+        "name": "Ultima",
```

Comment:
Ultima should be mode 4 with 20-second cooldown. Mode 2 is for cooldown items with unlimited uses.